### ISSUE TITLE
Make igm=None the default for spec.get_fnu

### ISSUE THREAD

**Issue by virajpandya** [flaresimulations/synthesizer#640]
Chris Lovell suggested posting this during an in-person hack in NYC.

**Is your feature request related to a problem? Please describe.**

Suppose a galaxy is instantiated with a 2D SFZH array:

```
arr_stars = Stars(log10ages=np.log10(tbins), metallicities=10**logZbins,,sfzh = SFH_grid)
galaxy = Galaxy(arr_stars)
spec = galaxy3stars.get_spectra_incident(grid)
```

Then if we want to get the spectrum flux density F_nu in nanoJy for the galaxy at z>0, we can just do 
```
spec.get_fnu(cosmo=Planck15,z=3.0,igm=None)
```

If we hadn't specified igm=None, then unaware users might not know that by default an IGM attenuation is applied to redshifted spectra.

**Describe the solution you'd like**
Maybe  make igm=None the default.

**Describe alternatives you've considered**

**Additional context**

**Comment by WillJRoper** [comment]
I feel like having an IGM should be the default but maybe the documentation should be better at stating that is the default. 

(Not that I have strong feelings. Happy for no IGM to become the new default but that’ll break a bunch of existing pipelines that assumes it’s there by default)

**Comment by christopherlovell** [comment]
I think given this should be as general as possible we should not assume any IGM attenuation. You may, for example, wish to look at the emission of local star clusters.

**Comment by WillJRoper** [comment]
Fair point! Feel free to raise a PR. If I get a chance before one is raised I'll make the change.

**Comment by christopherlovell** [comment]
fixed in #642
